app/models/logstash_queries/module_sparkline_query.rb

```diff
@@ -26,7 +28,7 @@ def top_level_histogram_agg(json)
       json.agg do
         json.terms do
           json.field 'modules'
-          json.size 0
+          json.size 100
```

Comment:
`"size": 0` used to specify that the aggregations include all possible buckets. That was [removed](https://github.com/elastic/elasticsearch/issues/18838), so aggregations must now specify a `size` as the maximum # of buckets to return. This particular query aggregates on the `modules` field. We currently have about 20 modules, so an upward limit of 100 should be plenty even if we add some additional modules later.